ai/copilot-extension.md

```diff
@@ -54,8 +54,8 @@ To use the Telerik MAUI Copilot extension:
 
 The following examples demonstrate useful prompts for the Telerik MAUI extension:
 
-* "`@telerikmaui` Generate a DataGrid with sorting and paging enabled."
-* "`@telerikmaui` Create a Telerik ComboBox for MAUI with multiple selection enabled."
+* "`@telerikmaui` Generate a DataGrid with 5 columns and 500 records. Enable sorting for the firs column and include paging."
+* "`@telerikmaui` Create a Telerik ComboBox with sample data for customers. Enable multiple selection and preselect the first two customers."
```

Comment:
Shouldn't the prompts be very basic/getting started. Is the pre-selection a valid basic scenario?